src/Microsoft.AspNetCore.Razor.VSCode/src/TelemetryReporter.ts

```diff
@@ -74,7 +73,6 @@ export class TelemetryReporter {
             {
                 path: project.path,
                 configurationName,
-                targetFramework,
```

Comment:
FYI @akshita31 we'll need to coordinate your telemetry work with our next release (mid April maybe?). I'm removing the `TargetFramework` reporting from our side.

Reply:
@NTaylorMullen I am trying to fit that into our next release(sometime around next week), so we should be good